lms/static/css/vendor/responsive-carousel/responsive-carousel.css

```diff
@@ -1,20 +0,0 @@
-/*
- * responsive-carousel
- * https://github.com/filamentgroup/responsive-carousel
- *
- * Copyright (c) 2012 Filament Group, Inc.
- * Licensed under the MIT, GPL licenses.
- */
-.carousel {
```

Comment:
I'm wondering if this carousel CSS actually still is in use. I noticed some uses of class carousel in checkout_receipt.html, incourse_reverify.html, pay_and_verify.html, and reverify.html.

There is also a small amount of carousel CSS in https://github.com/edx/edx-platform/blob/master/lms/static/sass/views/_verification.scss#L603.